### ISSUE TITLE
NTR: HMG-CoA lyase low in fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8196]
New term request HMG-CoA lyase low in fibroblasts (segal_070726184935): 3-hydroxy-3-methylglutaryl-CoA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000216
- label: Reduced HMG-CoA lyase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:28583327
- def: Acitivity of 3-hydroxy-3-methylglutaryl coenzyme A lyase (HMG CoA lyase) below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8196
- synonyms:
